index.html

```diff
@@ -154,9 +154,9 @@ <h1>Languages</h1>
           </li>
         </ul>
         <h2>Frameworks</h2>
-        <img src="/Asset/arrow.svg" class="arrow1" alt="" />
+        <img src="/Asset/arrowr.svg" class="arrow1" alt="" />
         <h3>Skills</h3>
-        <img src="/Asset/arrow.svg" class="arrow2" alt="" />
+        <img src="/Asset/arrowr.svg" class="arrow2" alt="" />
```

Comment:
[OPTIONAL] 
- [ ] Kindly use `ul` tag for technology categories ("Languages", "Frameworks" and "Skills") and `li` tag for each of the category. To follow correct HTML semantic. 
This is a requirement of the project but the first reviewer didn't notice it. So I requested this as an optional but I highly recommend you to make the change